packages/flutter_tools/lib/src/device.dart

```diff
@@ -771,6 +772,7 @@ class DebuggingOptions {
       this.traceAllowlist,
       this.enableImpeller = false,
       this.uninstallFirst = false,
+      this.enableDartProfiling = true,
```

Comment:
This could just as well be `false`, but `true` allows the default to retain the current behavior.